templates/layout.html

```diff
@@ -67,7 +67,7 @@
                     </ul>
                 </li>
                 <li class="depth-1">
-                    <a href="https://muncssociety.slack.com/">Slack Community</a>
+                    <a href="https://discord.muncssociety.com/">Discord Guild</a>
```

Comment:
Domain name is muncompsci.ca

Reply:
3ac1c545e6a20ff3574be86855797f3b6117c821

Reply:
Nope, still wrong

Reply:
i am good code

Reply:
648374bc6717a229849c6911603ce3219fe038c3